server/routes.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const db = require('../database/controllers');
+
+const router = express.Router();
+
+const queryAll = (req, res) => {
+  const { song } = req.params;
+  let requestEnd = req.originalUrl.split('/');
+  requestEnd = requestEnd[requestEnd.length - 1];
+  const dbQuery = (requestEnd === 'comments') ? db.getComments : db.getSong;
+
+  dbQuery(song)
+    .then((dbFind) => {
+      const returnData = (requestEnd === 'comments') ? dbFind : dbFind[0];
+      res.status(200).send(JSON.stringify({ data: returnData }));
```

Comment:
dbFind[0] is the first index in the array returned by the query right? 
suggestion to make it a variable to be more clear

Reply:
Changed it from dbFind to dbData